docker/build/README.md

```diff
@@ -0,0 +1,20 @@
+### Docker image for building TubeMQ
+#### pull
+- for all contributors
+```
+docker pull tdbank/tubemq-build
```

Comment:
Can we replace inner concept "tdbank" with some generalduty concept?

Reply:
apachetubemq/tubemq-build ?